realtime/constants.js

```diff
@@ -37,6 +39,12 @@ module.exports = {
       del: 'refocus.internal.realtime.subject.remove',
     },
 
+    room: {
+      add: 'imc.internal.realtime.room.add',
```

Comment:
is the "imc" name surviving the jump into refocus? i think these should be refocus.internal.realtime.room...